### DISCUSSION TITLE
Add an ability to set custom working directory

### DISCUSSION THREAD

**Discussion by kuncevic** [BetaHuhn/deploy-to-vercel-action#158] [Ideas] [answered]
[Vercel Deploy Action](https://github.com/ngduc/vercel-deploy-action) supports [working-directory ](https://github.com/ngduc/vercel-deploy-action/blob/master/.github/workflows/example-angular.yml#L42) option, that allow to set an output folder. 

In my case I do have multiple projects in my monorepo so each project has a different output folder.

It would be great if `deploy-to-vercel-action` can support such option too.

**Comment by BetaHuhn** [comment, marked as answer]
Hi @kuncevic,

this is now available in [v1.8.0](https://github.com/BetaHuhn/deploy-to-vercel-action/releases/tag/v1.8.0)! You can use the `WORKING_DIRECTORY` input to specify the working directory for the Vercel CLI.

Let me know if it works!
